examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -49,15 +50,16 @@ import java.net.URI
 import kotlinx.android.synthetic.main.activity_trip_service.mapView
 import kotlinx.android.synthetic.main.bottom_sheet_faster_route.*
 import kotlinx.android.synthetic.main.content_simple_mapbox_navigation.*
+import kotlinx.coroutines.channels.Channel
 import timber.log.Timber
 
 class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
 
     private val startTimeInMillis = 5000L
     private val countdownInterval = 10L
     private val maxProgress = startTimeInMillis / countdownInterval
-    private val locationEngineCallback =
-        MyLocationEngineCallback(this)
+    private val locationEngineCallback = MyLocationEngineCallback(this)
+    private val restartSessionEventChannel = Channel<RestartTripSessionAction>(1)
```

Comment:
Should we `cancel` / `close` the `Channel` in `onDestroy`?

Reply:
Done.